provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/TpmRegistrationResult.java

```diff
@@ -0,0 +1,18 @@
+/*
+ *
+ *  Copyright (c) Microsoft. All rights reserved.
+ *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ *
+ */
+
+package com.microsoft.azure.sdk.iot.provisioning.device;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class TpmRegistrationResult
+{
+    @Getter
+    protected String authenticationKey;
```

Comment:
Add some javadocs explaining what this field is. Maybe link to an MS doc page on TPM authentication for further explanation as well.